README.md

```diff
@@ -1,15 +1,60 @@
-# SSWG Logging API
 
-* Proposal: SSWG-xxxx
+# Proposal Review: SSWG-0001 (Server Logging API)
+
+After the great [discussion thread](https://forums.swift.org/t/server-logging-api/18834), we are proposing this as a final revision of this proposal and enter the proposal review phase which will run until the 20th January 2019.
+
+We have integrated most of the feedback from the discussion thread so even if you have read the previous version, you will find some changes that you hopefully agree with. To highlight a few of the major changes:
+
+- Logging meta-data is now structures and now supports nested dictionaries and list values, apart from strings of course.
+- only locally relevant meta-data can now be passed to the individual log methods.
+- ship a multiplex logging solution with the API package
+
+The feedback model will be very similar to the one known from Swift Evolution. The community is asked to provide feedback in the way outlined below and after the review period finishes, the SSWG Steering Committee will -- based on the community feedback -- decide whether to promote the proposal to the [Sandbox](https://github.com/swift-server/sswg/blob/master/process/incubation.md#process-diagram) maturity level or not.
```

Comment:
👍

Reply:
SSWG Steering Committee -> SSWG, we stopped using the term "Steering Committee" and only refer to it as SSWG now

Reply:
should this be Sandbox or Incubating? I would argue for the latter but we can also vote on that in the SSWG discussion

Reply:
@tomerd 
- hmm, but what do we call the SSWG community then, so everybody who's not on the ~~steering group~~?
- the process diagram only has a line from 'proposal' to 'sandbox' so I picked that :). Also I think 'incubating' requires some usage right?

Reply:
addressed